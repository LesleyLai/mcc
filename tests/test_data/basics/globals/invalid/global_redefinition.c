int x = 42;
int x = 42;

int main(void)
{
  int foo = 1;
  return foo + x;
}