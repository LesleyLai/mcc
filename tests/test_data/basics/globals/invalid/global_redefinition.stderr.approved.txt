{{filename}}:1:1: Error: redefinition of 'x'

